### PR TITLE
Readme for health monitor

### PR DESCRIPTION
**Pull request by souravgupta-msft**

### CODE REVIEW

File: tools/health-monitor/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Blobfuse2 Health Monitor (Preview)
+
+## About
+
+Blobfuse2 Health Monitor is a tool which will help in monitoring the mounts done via Blobfuse2. It supports the following types of monitors:
```
Comment (gapra-msft) [inline tools/health-monitor/README.md:5]:
```suggestion
Blobfuse2 Health Monitor is a tool which will help in monitoring Blobfuse2 mounts. It supports the following types of monitors:
```

File: tools/health-monitor/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+The different configuration options for the health monitor are,
+- `enable-monitoring: true|false`: Boolean parameter to enable health monitor. By default it is disabled
+- `stats-poll-interval-sec: <TIME IN SECONDS>`: Blobfuse2 stats polling interval (in sec). Default is 10 seconds
+- `process-monitor-interval-sec: <TIME IN SECONDS>`: CPU and memory usage polling interval (in sec). Default is 30 sec
+- `output-path: <PATH>`: Path where health monitor will generate its output file. It takes the current directory as default, if not specified. Output file name will be `monitor_<pid>.json`
+- `monitor-disable-list: <LIST OF MONITORS>`: List of monitors to be disabled. To disable a monitor, add its corresponding name in the list
+    - `blobfuse_stats` - Disable blobfuse2 stats polling
+    - `cpu_profiler` - Disable CPU monitoring on blobfuse2 process
+    - `memory_profiler` - Disable memory monitoring on blobfuse2 process
+    - `cache_monitor` - Disable file cache directory monitor
```
Comment (gapra-msft) [inline tools/health-monitor/README.md:34]: Should we explicitly rename this to file cache monitor? Cause in the future we could have a stream cache monitor or something right?

**Review by gapra-msft — APPROVED** [review]

File: tools/health-monitor/README.md (hunk 2 above)
Comment (vibhansa-msft) [reply tools/health-monitor/README.md:34]: we may not have stream cache monitor as such (as this is just monitoring a directory events and streaming will rather be part of the stats-monitoring), but still renaming this to file_cache_monitor makes more sense.

File: tools/health-monitor/README.md (hunk 1 above)
Comment (vibhansa-msft) [inline tools/health-monitor/README.md:5]: We can mention here, there will one instance of health-monitor running per mount (if configured)

**Review by vibhansa-msft — APPROVED** [review]

File: tools/health-monitor/README.md (hunk 1 above)
Comment (souravgupta-msft) [reply tools/health-monitor/README.md]: Added this note.
> **Note:** Health Monitor runs as a separate process where one health monitor process is associated with monitoring one blobfuse2 mounted directory.

File: tools/health-monitor/README.md (hunk 2 above)
Comment (souravgupta-msft) [reply tools/health-monitor/README.md]: Renamed to file_cache_monitor

**Pull request merged**